torch/_refs/__init__.py

```diff
@@ -1972,6 +1973,26 @@ def _reduction(
     return result
 
 
+def _make_copy_from_view(fn):
+    # Does not handle multiple outputs, but I think we don't have any such an op
+    name = fn.__name__
+    fn = out_wrapper()(fn)
```

Comment:
Let's be very clear about why this applies the out wrapper instead of asking the developer to do it

Reply:
In fact, I believe that this application of `out_wrapper` should not be necessary. I should delete it and simply reimplement the logic, as I've done below.

Reply:
(offline discussion between @mruberry and @lezcano suggested a tweak to this idea)